cheroot/workers/threadpool.py

```diff
@@ -92,6 +93,53 @@ def __init__(self, server):
         }
         threading.Thread.__init__(self)
 
+    def log_start_stats(self):
+        if self.server.stats['Enabled']:
+            self.start_time = time.time()
+
+    def log_close_stats(self):
+        if self.server.stats['Enabled']:
+            self.requests_seen += self.conn.requests_seen
+            self.bytes_read += self.conn.rfile.bytes_read
+            self.bytes_written += self.conn.wfile.bytes_written
+            self.work_time += time.time() - self.start_time
+            self.start_time = None
+
+    def conn_expired(self, conn, last_active, cur_time):
+        srv_timeout = self.server.timeout
+        return cur_time - last_active > srv_timeout
+
+    def get_expired_conns(self, conn_socks, cur_time):
```

Comment:
I think that creating `cur_time` somewhere separate from actual timeout check is wrong. It should go to `conn_expired` check.